SPECIFICATION.md

```diff
@@ -1,179 +1,175 @@
 # Ceramic Protocol Specification
 [![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)](https://lbesson.mit-license.org/)
 [![](https://img.shields.io/badge/Chat%20on-Discord-orange.svg?style=flat)](https://discord.gg/6VRZpGP)
-[![Twitter](https://img.shields.io/twitter/follow/ceramicnetwork?label=Follow&style=social)](https://twitter.com/ceramicnetwork) 
+[![Twitter](https://img.shields.io/twitter/follow/ceramicnetwork?label=Follow&style=social)](https://twitter.com/ceramicnetwork)
 
 This document contains the technical specification for the Ceramic protocol. For a higher-level introduction to the Ceramic network, see the [Ceramic Overview](https://github.com/ceramicnetwork/ceramic/blob/master/OVERVIEW.md).
 
 ## Index
 
 - [Protocol overview](#protocol-overview)
-- [Document lifecycle](#document-lifecycle)
-  - [Creating a document](#creating-a-document)
-  - [Updating a document](#updating-a-document)
-  - [Looking up a document](#looking-up-a-document)
-- [Document Identifiers](#document-identifiers)
-- [Document log](#document-log)
+- [Stream lifecycle](#stream-lifecycle)
+  - [Creating a stream](#creating-a-stream)
+  - [Updating a stream](#updating-a-stream)
+  - [Looking up a stream](#looking-up-a-stream)
+- [Stream Identifiers](#stream-identifiers)
+- [Stream log](#stream-log)
```

Comment:
I tend to call this "Commit log" when talking about it. I think it would make sense to change it here.